docs/section5/Rule5-48.md

```diff
@@ -15,37 +15,26 @@
 **Data Lookup:** None  
 **Function Call:** 
 
-  1. get_zone_conditioning_category()
-  2. match_data_element()
+  1. match_data_element()
 
 ## Rule Logic:  
 
-- Get zone conditioning category for B_RMR: `zcc_b = get_zone_conditioning_category(B_RMR)`
-
-- Get zone conditioning category for P_RMR: `zcc_p = get_zone_conditioning_category(P_RMR)`
-
```

Comment:
Per above, we do need to get zone conditioning category for B-RMR, but not for P-RMR.

Reply:
This is now added.